.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,52 @@
+<!-- NAMING YOUR ISSUE: Please prefix your issue with the name of the sub-project -->
```

Comment:
Does this make sense for cucumber-rails since it is not in the monorepo? I'm confused what to put in a PR I'm opening at the moment.

Reply:
Good point. I'll remove this. :)

Reply:
I've just created a PR to fix this, but I'm not 100% sure about the new phrasing. Any thoughts?
https://github.com/cucumber/cucumber-rails/pull/374

Reply:
Seems fine to me. :+1: